tests/apps/test_rf_ram.py

```diff
@@ -0,0 +1,12 @@
+import struct
+import time
+
+
+class TestRfRam:
+    '''Tests for Vault app.'''
+
+    def test_access_rf_ram(self, app, stop_app):
```

Comment:
Same here, `TestRamPage` and `test_access_ram_page`, and rename the file to `test_ram_page.py` :)